README.md

```diff
@@ -12,50 +12,54 @@ Given the recent (and developing) situation with decision in the .net core effor
 # Examples
 ### Finding a Store
 ```C#
+var api = new DominosApi();
```

Comment:
Since this is example code, perhaps here and elsewhere in the README we should do:

```C#
IDominoesApi api = new DominosApi();
```

It may direct people towards the IDominoesApi interface, which has additional comments explaining how to use these methods.